on logout user is now navigated to login page

lazarpe opened the pull request.

lazarpe committed:
on logout user is now navigated to login page

davidemarcoli reviewed (approved):
Looks good to me! But removing the Token from Localstorage and then reloading should be sufficient!

davidemarcoli committed:
Replace location.href with angular router
Improve canActivate method in the auth-guard.service.ts so that the user can't navigate to a page if he isn't signed in

davidemarcoli committed:
Fix deprecated JWT method

lazarpe committed:
Removed commented code.

Sashimi4 reviewed (approved):
LGTM
